### PR TITLE
Test/lars janno

### PR DESCRIPTION
**Pull request by jlunenburg**

### CODE REVIEW

File: challenge_restaurant/README.md [hunk 1]
```diff
@@ -3,28 +3,12 @@
 Responsible: Rein
```
Comment (MatthijsBurgh) [inline challenge_restaurant/README.md]: Maintainer is Ramon, please also remove year.

File: challenge_restaurant/README.md [hunk 2]
```diff
@@ -3,28 +3,12 @@
...
 
 ## Startup
+```
+* amigo-start should already be running
```
Comment (MatthijsBurgh) [inline challenge_restaurant/README.md]: Place commands between three graves.

File: challenge_set_a_table/README.md [hunk 3]
```diff
@@ -2,9 +2,14 @@
...
+* Put the robot in front of the door (the usual initial pose)
+* Make sure one of the combos is on the cabinet (look at the RoboCup knowledge for the possibilities). N.B., only the 'easy' items are relevant.
+* When the robot asks what to do, answer 'Set the table'. The rest should speak for itself.
+
 Launch procedure:
 ```
-amigo-start
+amigo-start (usually already running)
 amigo-free-mode
 rosrun challenge_set_a_table challenge_set_a_table.py amigo
```
Comment (MatthijsBurgh) [inline challenge_set_a_table/README.md]: Why not create a launch file?

File: challenge_restaurant/README.md [hunk 4]
```diff
@@ -3,28 +3,12 @@
...
+* make sure the Jetson is running (check if <robot-name>/pose_detector/openpose is running)
+* amigo-challenge restaurant (starts the middleware)
+* navigate around for a bit to get a proper gmap
+* to get the robot to take the order, answer "<robot-name> take the order"
+* when ordering, you can choose a beverage or a combo (food & food). Check the RoboCup knowledge to see what foods and drinks are.
```
Comment (MatthijsBurgh) [inline challenge_restaurant/README.md]: food & food? Not Food&beverage?

File: challenge_restaurant/README.md (hunk 1 above)
Comment (jlunenburg) [reply challenge_restaurant/README.md]: Fixed in 9ecad01e3839e2c035db397853fbd39948c1c2ba

File: challenge_restaurant/README.md (hunk 2 above)
Comment (jlunenburg) [reply challenge_restaurant/README.md]: Fixed in d34e371aa795efb2a8317025b878fcc08ceb7968

File: challenge_restaurant/README.md (hunk 4 above)
Comment (jlunenburg) [reply challenge_restaurant/README.md]: Yep, a combo is food & food

File: challenge_set_a_table/README.md (hunk 3 above)
Comment (LarsJanssenTUe) [reply challenge_set_a_table/README.md]: Updated in 90e90818e4d3ab3b0d62c0e93b6e6fe88aa18d45

**Pull request merged**